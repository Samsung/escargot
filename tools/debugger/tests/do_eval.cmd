b do_eval.js:7
c
eval a
e b
e c
e d
e e
print a
p b
p c
p d
p e
c
