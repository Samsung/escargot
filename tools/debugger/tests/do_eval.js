let a = 42;
let b = "Alma";
let c = undefined;
let d = function(){ return 1; }
let e = {a:"a", b:10};

print(a);
