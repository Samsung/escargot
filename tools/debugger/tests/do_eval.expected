Connecting to: localhost:6501
Connection created!!!
Stopped at tools/debugger/tests/do_eval.js:2
(escargot-debugger) b do_eval.js:7
Breakpoint 1 at tools/debugger/tests/do_eval.js:7
(escargot-debugger) c
Stopped at breakpoint:1 tools/debugger/tests/do_eval.js:7
(escargot-debugger) eval a
42(escargot-debugger) e b
Alma(escargot-debugger) e c
undefined(escargot-debugger) e d
function(){ return 1; }(escargot-debugger) e e
[object Object](escargot-debugger) print a
42(escargot-debugger) p b
Alma(escargot-debugger) p c
undefined(escargot-debugger) p d
function(){ return 1; }(escargot-debugger) p e
[object Object](escargot-debugger) e k
Exception: ReferenceError: k is not defined(escargot-debugger) p k
Exception: ReferenceError: k is not defined(escargot-debugger) c
Print: 42
Connection closed.
